xlineapi/src/lib.rs

```diff
@@ -235,124 +254,35 @@ impl User {
 }
 
 /// Wrapper for requests
-#[derive(Serialize, Deserialize, Debug, Clone)]
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
```

Comment:
What's the `PartialEq` for?

Reply:
The `Command` derives `PartialEq` that requires its member `RequestWithToken` to also derive `PartialEq`. 

There's a problem that the conditional derive using `cfg_attr` doesn't seems to work with nested derives, so this derive seems need to be retained.